### PR TITLE
[Nezha P1] Add loss, return_dict, hidden and attention outputs

### PR DESCRIPTION
**Pull request by sijunhe**

### CODE REVIEW

File: paddlenlp/transformers/nezha/modeling.py [hunk 1]
```diff
@@ -433,12 +457,25 @@ def forward(self, input_ids, token_type_ids=None, attention_mask=None):
...
-        encoder_outputs, _ = self.encoder(embedding_output, extended_attention_mask)
+        encoder_hidden_outputs, encoder_att_outputs = self.encoder(embedding_output, extended_attention_mask)
 
-        sequence_output = encoder_outputs[-1]
+        sequence_output = encoder_hidden_outputs[-1]
         pooled_output = self.pooler(sequence_output)
 
-        return sequence_output, pooled_output
+        if not return_dict:
+            return (sequence_output, pooled_output, encoder_att_outputs)
```
Comment (wj-Mcat) [inline paddlenlp/transformers/nezha/modeling.py]: 你的这里还没有处理`output_hidden_states`和`output_attentions`的逻辑，而是强制返回 attention。这里的逻辑处理可参考：https://github.com/PaddlePaddle/PaddleNLP/blob/develop/paddlenlp/transformers/bert/modeling.py#L483

**Review by wj-Mcat — COMMENTED** [review]
就一个小地方有些疑问，其它 lgtm

**Review by wj-Mcat — APPROVED** [review]
LGTM
